### PR TITLE
Three dot log docs

### PR DESCRIPTION
**Pull request by tbantle22**

### CODE REVIEW

File: content/reference/sql/version-control/dolt-sql-functions.md [hunk 1]
```diff
@@ -417,12 +417,16 @@ The `DOLT_LOG()` table function takes up to two optional revision arguments:
 
 - `optional_revision`: a branch name, tag, or commit ref (with or without an ancestor
   spec) that specifies which ancestor commits to include in the results. If no revisions
-  are specified, the default is the current branch `HEAD`. If you'd like to get [two dot
-  logs](https://matthew-brett.github.io/pydagogue/git_log_dots.html) (all commits
-  reachable by `revision2`, but NOT reachable by `revision1`), you can use `..` between
-  revisions (`DOLT_LOG('revision1..revision2')`) or `^` in front of the revision you'd
-  like to exclude (`DOLT_LOG('revision2', '^revision1')`). Note: if providing two
+  are specified, the default is the current branch `HEAD`. 
+    - If you'd like to get [two dot logs]([https://matthew-brett.github.io/pydagogue/git_log_dots.html](https://matthew-brett.github.io/pydagogue/git_log_dots.html#logging-with-two-dots))
```
Comment (fulghum) [inline content/reference/sql/version-control/dolt-sql-functions.md]: Is this valid markdown syntax for links? When I look at the GitHub rendered version, I don't see these links showing up.

File: content/reference/sql/version-control/dolt-sql-functions.md [hunk 2]
```diff
@@ -417,12 +417,16 @@ The `DOLT_LOG()` table function takes up to two optional revision arguments:
...
-  reachable by `revision2`, but NOT reachable by `revision1`), you can use `..` between
-  revisions (`DOLT_LOG('revision1..revision2')`) or `^` in front of the revision you'd
-  like to exclude (`DOLT_LOG('revision2', '^revision1')`). Note: if providing two
+  are specified, the default is the current branch `HEAD`. 
+    - If you'd like to get [two dot logs]([https://matthew-brett.github.io/pydagogue/git_log_dots.html](https://matthew-brett.github.io/pydagogue/git_log_dots.html#logging-with-two-dots)) 
+      (all commits reachable by `revision2`, but NOT reachable by `revision1`), you can 
+      use `..` between revisions (`DOLT_LOG('revision1..revision2')`) or `^` in front of 
+      the revision you'd like to exclude (`DOLT_LOG('revision2', '^revision1')`). Note: if providing two
   revisions, one must contain `^`.
+    - If you'd like to get [three dot logs]([https://matthew-brett.github.io/pydagogue/git_log_dots.html#logging-with-two-dots](https://matthew-brett.github.io/pydagogue/git_log_dots.html#logging-with-three-dots))
```
Comment (fulghum) [inline content/reference/sql/version-control/dolt-sql-functions.md]: ditto for this link, too.

**Review by fulghum — APPROVED** [review]
Content looks good. I think you just need to fix the markdown link syntax in two places.

File: content/reference/sql/version-control/dolt-sql-functions.md (hunk 1 above)
Comment (tbantle22) [reply content/reference/sql/version-control/dolt-sql-functions.md]: Oops no thanks

**Pull request merged**